apiary.apib

```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
+            <td>isAggregationRequired</td>
+        </tr>
+        <tr>
+            <td>type</td>
+            <td>Type</td>
+        </tr>
+        <tr>
+            <td>businessNumber</td>
+            <td>Enterprise Competitor Business Number</td>
+        </tr>
+        <tr>
+            <td>businessAlreadyPresent</td>
+            <td>businessAlreadyPresent</td>
+        </tr>
+        <tr>
+            <td>sfdcAccountId</td>
+            <td>sfdcAccountId</td>
+        </tr>
+         <tr>
+            <td>sfdcOpportunityId</td>
+            <td>sfdcOpportunityId</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Error Codes</caption>
+        <thead>
+        <tr>
+            <th>HTTP Status</th>
+            <th>BirdEye error code</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1161</td>
+            <td>Invalid API key</td>
+        </tr>
+        <tr>
+            <td>429 Too Many Requests</td>
+            <td>89</td>
+            <td>Rate limit exceeded</td>
+        </tr>
+        <tr>
+            <td>404 Not Found</td>
+            <td>1011</td>
+            <td>Business id is invalid</td>
+        </tr>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1167</td>
+            <td>API key is missing</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
++ Request (application/json)
+
+    + Headers
+
+            Accept: application/json
+
++ Response 200
+
+    + Body
+
+{
+    "competitorId": 100037739,
+    "templateid": 0,
+    "isAggregationRequired": 1,
+    "type": "Enterprise-Location-Competitor",
+    "businessNumber": 161295237142347,
+    "businessAlreadyPresent": false,
+    "sfdcAccountId": null,
+    "sfdcOpportunityId": null
+}
+
++ Response 401
+
+    [1161][]
+
++ Response 429
+
+    [89][]
+
++ Response 404
+
+    [1011][]
+
++ Response 401
+
+    [1167][]    
+    
+## Create Child Business [/v1/signup/competitor/{business_number}{?api_key}]
+New child business can be created in competitor enterprise:
+
++ Parameters
+
+    + business_number (string, required) ... the business ID
+    + api_key (string, required) ... the API key
+
+### Create New Child Business in Competitor Enterprise [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+    
+    curl -H "content-type: application/json" -H "accept:application/json" -X POST -d '{"businessName":"Location_compca","userEmailId":"rhonda@reviewmanagement.com","countryCode":"CA","phone":"408-xxx-xxxx","zip":"95035","competitorId":100037693,"type":"Product"}' 
+    "https://api.birdeye.com/resources/v1/signup/competitor/151134205012586?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451" -v
+
+<center><b>Example Payload</b></center>
+
+    {"businessName": "Location_compca",
+    "userEmailId": "rhonda@reviewmanagement.com",
```

Comment:
@khyati-gupta use a different email address in this payload and the curl commands.